packages/packages/commonwell-sdk/src/models/organization.ts

```diff
@@ -53,8 +53,9 @@ export const organizationSchema = z.object({
       z.object({
         serviceType: z.string(),
         gatewayType: z.string(),
-        isAsync: z.boolean(),
-        gatewayTimeout: z.number(),
+        isAsync: z.boolean().optional(),
+        gatewayTimeout: z.number().optional(),
+        endpointLocation: z.string().optional(),
```

Comment:
This is the FHIR endpoint? If so, should we have a setting for each or, or a global one? We essentially only have one FHIR server and likely will keep it like this.

Reply:
Yae this is going to refer to the FHIR endpoint. Right now im just referencing the global env ```FHIR_SERVER_URL```